lib/common/bitstream.h

```diff
@@ -109,11 +109,11 @@ MEM_STATIC size_t BIT_closeCStream(BIT_CStream_t* bitC);
 *  bitStream decoding API (read backward)
 **********************************************/
 typedef struct {
-    size_t   bitContainer;
-    unsigned bitsConsumed;
     const char* ptr;
     const char* start;
     const char* limitPtr;
+    size_t   bitContainer;
+    unsigned bitsConsumed;
```

Comment:
On x86 clang-9.0.1 I measured a 4% decompression speed boost from this change alone. It was neutral for gcc.

This one is very interesting, I will look into it further.